spec/models/outgoing_message_spec.rb

```diff
@@ -143,12 +143,45 @@
     expect(@outgoing_message.get_text_for_indexing).not_to include("foo@bar.com")
   end
 
-  it "should not display email addresses on page" do
-    expect(@outgoing_message.get_body_for_html_display).not_to include("foo@bar.com")
-  end
+  context "when formatting the message for html display" do
+
+    it "does not display email addresses on page" do
+      expect(@outgoing_message.get_body_for_html_display).not_to include("foo@bar.com")
+    end
+
+    it "links to help page where email address was" do
+      expect(@outgoing_message.get_body_for_html_display).to include('<a href="/help/officers#mobiles">')
+    end
+
+    it "does not force long lines to wrap" do
+      long_line = "long string of 125 characters, set so the old line break " \
+                  "falls here, and making sure even longer lines aren't " \
+                  "affected either"
+      @outgoing_message.body = long_line
+      expect(@outgoing_message.get_body_for_html_display).not_to include('<br>')
```

Comment:
What about `\n`s? I'd maybe make this:

```ruby
expect(@outgoing_message.get_body_for_html_display).to eq(long_line)
```

I don't know – on one hand its a bit more explicit that we're saying we don't want the two specific breaking characters. On the other, I like that we're stating that the string should be unchanged.

Any thoughts @crowbot?